rocketpy/simulation/monte_carlo.py

```diff
@@ -104,6 +113,17 @@ def __init__(
             `out_of_rail_stability_margin`, `out_of_rail_time`,
             `out_of_rail_velocity`, `max_mach_number`, `frontal_surface_wind`,
             `lateral_surface_wind`. Default is None.
+        data_collector : dict, optional
+            A dictionary whose keys are the names of the exported variables
+            and the values are callback functions. The keys (variable names) must not
+            overwrite the default names on 'export_list'. The callback functions receive
+            a Flight object and returns a value of that variable. For instance
+
+            .. code-block:: python
+                custom_data_collector = {
+                    "max_acceleration": lambda flight: max(flight.acceleration(flight.time)),
+                    "date": lambda flight: flight.env.date,
+                }
```

Comment:
I wonder how this code block is being rendered on our docs page

Reply:
Docs are breaking. @Lucas-Prates 

https://readthedocs.org/projects/rocketpyalpha/builds/26380046/

Reply:
fixed